modules/twinklespeedy.js

```diff
@@ -2042,28 +2045,14 @@ Twinkle.speedy.callback.evaluateSysop = function twinklespeedyCallbackEvaluateSy
 		}
 	});
 
-	var warnusertalk = false;
-	if (form.warnusertalk.checked) {
-		$.each(normalizeds, function(index, norm) {
-			if (Twinkle.getPref('warnUserOnSpeedyDelete').indexOf(norm) !== -1) {
-				if (norm === 'g6' && values[index] !== 'copypaste') {
-					return true;
-				}
-				warnusertalk = true;
-				return false;  // break
-			}
-		});
-	}
+	var warnusertalk = form.warnusertalk.checked && normalizeds.some(function (norm, index) {
```

Comment:
Nice.  I always forget that `.some` and `.every` are available, and lump them in with `.includes()`